dask/array/tests/test_random.py

```diff
@@ -127,110 +145,173 @@ def test_random_seed():
-    da.random.uniform(size=5, chunks=3).compute()
-    da.random.vonmises(2, 3, size=5, chunks=3).compute()
-    da.random.wald(1, 2, size=5, chunks=3).compute()
-
-    da.random.weibull(2, size=5, chunks=3).compute()
-    da.random.zipf(2, size=5, chunks=3).compute()
-
-    da.random.standard_cauchy(size=5, chunks=3).compute()
-    da.random.standard_exponential(size=5, chunks=3).compute()
-    da.random.standard_gamma(2, size=5, chunks=3).compute()
-    da.random.standard_normal(size=5, chunks=3).compute()
-    da.random.standard_t(2, size=5, chunks=3).compute()
+@pytest.mark.parametrize("sz", [None, 5, (2, 2)], ids=type)
+def test_random_all(sz):
+    da.random.beta(1, 2, size=sz, chunks=3).compute()
+    da.random.binomial(10, 0.5, size=sz, chunks=3).compute()
+    da.random.chisquare(1, size=sz, chunks=3).compute()
+    da.random.exponential(1, size=sz, chunks=3).compute()
+    da.random.f(1, 2, size=sz, chunks=3).compute()
+    da.random.gamma(5, 1, size=sz, chunks=3).compute()
+    da.random.geometric(1, size=sz, chunks=3).compute()
+    da.random.gumbel(1, size=sz, chunks=3).compute()
+    da.random.hypergeometric(1, 2, 3, size=sz, chunks=3).compute()
+    da.random.integers(5, high=15, size=sz, chunks=3).compute()
+    da.random.laplace(size=sz, chunks=3).compute()
+    da.random.logistic(size=sz, chunks=3).compute()
+    da.random.lognormal(size=sz, chunks=3).compute()
+    da.random.logseries(0.5, size=sz, chunks=3).compute()
+    da.random.multinomial(20, [1 / 6.0] * 6, size=sz, chunks=3).compute()
+    da.random.multivariate_hypergeometric([16, 8, 4], 6, size=sz, chunks=6).compute()
+    da.random.negative_binomial(5, 0.5, size=sz, chunks=3).compute()
+    da.random.noncentral_chisquare(2, 2, size=sz, chunks=3).compute()
+
+    da.random.noncentral_f(2, 2, 3, size=sz, chunks=3).compute()
+    da.random.normal(2, 2, size=sz, chunks=3).compute()
+    da.random.pareto(1, size=sz, chunks=3).compute()
+    da.random.poisson(size=sz, chunks=3).compute()
+
+    da.random.power(1, size=sz, chunks=3).compute()
+    da.random.rayleigh(size=sz, chunks=3).compute()
+
+    da.random.triangular(1, 2, 3, size=sz, chunks=3).compute()
+    da.random.uniform(size=sz, chunks=3).compute()
+    da.random.vonmises(2, 3, size=sz, chunks=3).compute()
+    da.random.wald(1, 2, size=sz, chunks=3).compute()
+
+    da.random.weibull(2, size=sz, chunks=3).compute()
+    da.random.zipf(2, size=sz, chunks=3).compute()
+
+    da.random.standard_cauchy(size=sz, chunks=3).compute()
+    da.random.standard_exponential(size=sz, chunks=3).compute()
+    da.random.standard_gamma(2, size=sz, chunks=3).compute()
+    da.random.standard_normal(size=sz, chunks=3).compute()
+    da.random.standard_t(2, size=sz, chunks=3).compute()
+
+
+def test_RandomState_only_funcs():
+    da.random.randint(10, size=5, chunks=3).compute()
+    with pytest.raises(DeprecationWarning):
+        da.random.random_integers(10, size=5, chunks=3).compute()
```

Comment:
What warning are you expecting here? I don't get any deprecation and I don't know what it should be since there's no `match`.

Reply:
`random_integers` was deprecated in numpy-1.11.0. So, you should be getting a `DeprecationWarning` with any higher version. I am getting one with numpy-1.24.2

Reply:
Ah, I see, thank you. I've found that someone had added an overly broad `-Wignore` to the package, which broke this test. I have removed that and the downstream build is working. Sorry for the noise here.